api/src/test/java/org/pmiops/workbench/workspaceadmin/WorkspaceAdminServiceTest.java

```diff
@@ -110,7 +110,6 @@
   @MockBean private FireCloudService mockFirecloudService;
   @MockBean private LeonardoNotebooksClient mockLeonardoNotebooksClient;
   @MockBean private LeonardoRuntimeAuditor mockLeonardoRuntimeAuditor;
-  @MockBean private MailService mockMailService;
```

Comment:
@jmthibault79 just keeping this change to remove a variable not being used

Reply:
sgtm